### PR TITLE
Read sheet

### PR DESCRIPTION
**Pull request by EmptyInfinity**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -227,6 +227,8 @@ You can find more information in the [Google Sheets API Documentation](https://d
 
 Action read spreadsheet. This action is based on [Google Spreadsheets API v4](https://developers.google.com/sheets/api/reference/rest/v4/spreadsheets/get). All data structures and limitations are the same to Google API.
 
+*Note*: Place your table it top left corner to data be processed in right way.
```
Comment (A3a3e1) [inline README.md:230]:
```suggestion
*Important!*: Place (start) your table in the top left corner (cell) for data to be processed in the right way. 
```

**Review by A3a3e1 — APPROVED** [review]

**Review by shulkaolka — CHANGES_REQUESTED** [review]

File: lib/helpers/utils.js [hunk 2]
```diff
@@ -66,3 +66,16 @@ module.exports.refreshToken = async (emitter, secretId) => {
 
 // eslint-disable-next-line max-len
 module.exports.sleep = delay => new Promise((resolve) => { setTimeout(resolve, delay || REQUEST_RETRY_DELAY); });
+
+module.exports.numToLetters = function numToSSColumnLetter(num) {
```
[the remark this answers is not in the document]
Comment (EmptyInfinity) [reply lib/helpers/utils.js:70]: But it's not presented in v3.0.0 and covered with tests

[the remark this answers is not in the document]
Comment (shulkaolka) [reply lib/helpers/utils.js:70]: https://github.com/elasticio/gspreadsheets/blob/add-faceless/lib/common.js#L1 present.
Please, use this function and cover test it

File: lib/helpers/utils.js [hunk 3]
```diff
@@ -6,6 +6,19 @@ const REQUEST_TIMEOUT = process.env.REQUEST_TIMEOUT || 10000; // 10s
 const REQUEST_MAX_RETRY = process.env.REQUEST_MAX_RETRY || 3;
 const REQUEST_RETRY_DELAY = process.env.REQUEST_RETRY_DELAY || 5000; // 5s
 
+module.exports.columnToLetter = function columnToLetter(column) {
```
Comment (shulkaolka) [inline lib/helpers/utils.js:9]: why you don want to use https://github.com/elasticio/gspreadsheets/blob/read-sheet-extend/lib/common.js#L1 ??

Comment (EmptyInfinity) [reply lib/helpers/utils.js:9]: I'm using it
I've just added module.exports to it

**Review by shulkaolka — APPROVED** [review]

**Pull request merged**